adminSiteClient/SuggestedChartRevisionListPage.tsx

```diff
@@ -0,0 +1,264 @@
+import * as React from "react"
+import { observer } from "mobx-react"
+import { observable, computed, action } from "mobx"
+import fuzzysort from "fuzzysort"
+import { Link } from "react-router-dom"
+
+import { TextField } from "./Forms"
+import { AdminLayout } from "./AdminLayout"
+import { uniq } from "../clientUtils/Util"
+import Select from "react-select"
+import { getStylesForTargetHeight } from "../clientUtils/react-select"
+import { highlight as fuzzyHighlight } from "../grapher/controls/FuzzySearch"
+import {
+    SuggestedChartRevisionList,
+    SuggestedChartRevisionListItem,
+} from "./SuggestedChartRevisionList"
+import { AdminAppContext, AdminAppContextType } from "./AdminAppContext"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { faSortAlphaDown } from "@fortawesome/free-solid-svg-icons/faSortAlphaDown"
+import { faSortAlphaUpAlt } from "@fortawesome/free-solid-svg-icons/faSortAlphaUpAlt"
+
+interface Searchable {
+    suggestedChartRevision: SuggestedChartRevisionListItem
+    term?: Fuzzysort.Prepared
+}
+
+@observer
+export class SuggestedChartRevisionListPage extends React.Component {
+    static contextType = AdminAppContext
+    context!: AdminAppContextType
+
+    @observable searchInput?: string
+    @observable maxVisibleCharts = 50
+    @observable suggestedChartRevisions: SuggestedChartRevisionListItem[] = []
+    @observable numTotalRows?: number
+
+    @observable sortBy: string = "updatedAt"
+    @observable sortDirection: string = "DESC"
```

Comment:
We already have a `SortOrder` type that would be good to use here: https://github.com/owid/owid-grapher/blob/06853e84ea16ac460ae6fed4d31081080b04f4a8/clientUtils/owidTypes.ts#L13-L16

Reply:
resolved in c9aedc1